cmake/faceliftMacros.cmake

```diff
@@ -714,6 +719,7 @@ function(facelift_add_test TEST_NAME)
     set_target_properties(${TEST_NAME} PROPERTIES
         LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/tests
         RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/tests
+        LINK_DEPENDS_NO_SHARED true
```

Comment:
We have set param to true inside "facelift_add_executable" above (line 717).

Reply:
Right, I've overlooked that one - will fix it and test it again. Thanks!

Reply:
Most probably it needs to be like that since the second set_target_properties will override the first one. We need to have it in both places. Most probably the second one should be replaced by set_property to append property instead of override all properties. I need to test new approach.

Reply:
Just tested it and lit doesn't override the first one which is a good news:)